guides/source/api_documentation_guidelines.md

```diff
@@ -60,11 +60,11 @@ end
 
 Communicate to the reader the current way of doing things, both explicitly and implicitly. Use the idioms recommended in edge. Reorder sections to emphasize favored approaches if needed, etc. The documentation should be a model for best practices and canonical, modern Rails usage.
 
-Documentation has to be concise but comprehensive. Explore and document edge cases. What happens if a module is anonymous? What if a collection is empty? What if an argument is nil?
+Documentation has to be concise. Explore and document edge cases. What happens if a module is anonymous? What if a collection is empty? What if an argument is nil?
```

Comment:
Doesn't "concise" mean "brief but comprehensive"? 🤔 ...

To me this would read "Documentation has to be brief but comprehensive but comprehensive."

Sounds like unnecessary repetition to me. No?

Reply:
In common usage, I find that "concise" emphasizes "brief" more than "comprehensive."  Here we want to especially emphasize "comprehensive" ("Explore and document edge cases..."), so I think the current way is fine.  But if it troubles you, we can change "concise" to "brief."